src/main/java/com/cybozu/kintone/client/module/parser/AppParser.java

```diff
@@ -10,9 +10,9 @@
 import java.lang.reflect.Type;
 import java.text.ParseException;
 import java.text.SimpleDateFormat;
+import java.time.ZoneOffset;
 import java.util.ArrayList;
 import java.util.Date;
-import java.util.Iterator;
 import java.util.TimeZone;
```

Comment:
@rshindo 
It seems like after the source code been refactored, java.util.TimeZone is not used anymore.
Could you please remove the import statement?

Thank you very much!